conf.py

```diff
@@ -197,6 +197,8 @@ def autoapi_prepare_jinja_env(jinja_env):
                     "ports/*/boards",
                     "ports/*/common-hal",
                     "ports/*/supervisor",
+                    "ports/analog/msdk",
+                    "ports/analog/peripherals",
```

Comment:
this frequently vexes me. perhaps we should explicitly list the documents we want with `include_patterns` instead (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-include_patterns)

Reply:
I like that idea.